code/game/gamemodes/miniantags/demons/slaughter demon/slaughter.dm

```diff
@@ -48,15 +48,18 @@
 
 /mob/living/simple_animal/demon/slaughter/proc/attempt_objectives()
 	if(mind)
-		to_chat(src, src.playstyle_string)
-		to_chat(src, "<B><span class ='notice'>You are not currently in the same plane of existence as the station. Use the blood crawl action at a blood pool to manifest.</span></B>")
+		var/list/messages = list()
+		messages.Add(playstyle_string)
+		messages.Add("<b><span class ='notice'>You are not currently in the same plane of existence as the station. Use the blood crawl action at a blood pool to manifest.</span></b>")
 		SEND_SOUND(src, sound('sound/misc/demon_dies.ogg'))
 		if(!vialspawned)
 			SSticker.mode.traitors |= mind
 			mind.add_mind_objective(/datum/objective/slaughter)
 			mind.add_mind_objective(/datum/objective/demon_fluff)
-			mind.announce_objectives(title = FALSE)
-		to_chat(src, "<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Slaughter_Demon)</span>")
+			messages.Add(mind.prepare_announce_objectives(FALSE))
```

Comment:
99% sure this will act strangely with vial-spawned demons/revenants

Reply:
need to test this, thanks for the heads up

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/2bdc7cd7-8047-4b52-8125-7e627660631f)
correct, this is bugged

Reply:
fix in latest commit:
![Uploading image.png…]()

Reply:
fix in latest commit:
![image](https://github.com/ParadiseSS13/Paradise/assets/12197162/e939cd66-bef2-4188-9469-358e39ca15ff)